src/main/java/com/epam/model/Reservation.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
I'm not sure, but no args/all args constructor is already defined in Data/Builder annotation?

Reply:
only @RequiredArgsConstructor https://projectlombok.org/features/Data